test/unit/test.rdf.js

```diff
@@ -365,13 +366,17 @@ describe("lib/rdf", function() {
       expect(locale.childNodes[3].childNodes[0].data).to.be.equal("名前");
       expect(locale.childNodes[5].tagName).to.be.equal("em:description");
       expect(locale.childNodes[5].childNodes[0].data).to.be.equal("紹介");
+      expect(locale.childNodes[7].tagName).to.be.equal("em:creator");
```

Comment:
I don't know, jpm generate `<em:creator/>` by default even you don't have `author` in package.json, this should be the problem with `formatAuthor(manifest.author)`, I think.

You can test it with test2.zip attached in https://github.com/mozilla-jetpack/jpm/pull/502#issuecomment-189050690 by remove the `author` flag, then you'll get
```js
          <em:unpack>false</em:unpack>
          <em:version>0.0.1</em:version>
          <em:name>My-addon</em:name>
          <em:creator/>
          <em:multiprocessCompatible>true</em:multiprocessCompatible>
```